### PR TITLE
Javascript/javascript2/week1

### PR DESCRIPTION
**Pull request by DeepthiDukka**

### CODE REVIEW

File: javascript/javascript2/week1/Shortestword/shortestWord.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+};
+let result = findShortestWord(danishWords);
+console.log(result);
+
+
+// Method 2 of finding the shortest word in a array
+function shortestWord(arr) {
+    let shortest = '';
+    // function to check the array elements length
+    arr.map(function(str){
```
Comment (flom84) [inline javascript/javascript2/week1/Shortestword/shortestWord.js]: i believe it's better to use `forEach` and not map. Remember that map always returns an array and you don't need that array

File: javascript/javascript2/week1/Shortestword/shortestWord.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Method 1 of finding the shortest word in an array
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium',[], 0, null, undefined, 'aa', Infinity, true];
+function findShortestWord(word) {
+let shortest = word[0];
+for(let i=0; i < word.length; i++){
+    if (typeof word[i] === 'string' &&
+        (shortest === ''|| word[i].length < shortest.length)) {
```
Comment (flom84) [inline javascript/javascript2/week1/Shortestword/shortestWord.js]: this should be sufficient
```suggestion
        (word[i].length < shortest.length)) {
```

File: javascript/javascript2/week1/Shortestword/shortestWord.js (hunk 1 above)
Comment (DeepthiDukka) [reply javascript/javascript2/week1/Shortestword/shortestWord.js]: implemented the given feedback.

File: javascript/javascript2/week1/Shortestword/shortestWord.js (hunk 2 above)
Comment (DeepthiDukka) [reply javascript/javascript2/week1/Shortestword/shortestWord.js]: corrected the code.

**Comment by DeepthiDukka** [comment]
closed this request after feedback is fixed as the later branches are already merged.

**Pull request closed without merging**